s3-source-connector/build.gradle.kts

```diff
@@ -21,7 +21,7 @@ plugins { id("aiven-apache-kafka-connectors-all.java-conventions") }
 val amazonS3Version by extra("1.12.729")
 val amazonSTSVersion by extra("1.12.729")
 val s3mockVersion by extra("0.2.6")
-val parquetVersion by extra("1.14.3")
+val parquetVersion by extra("1.11.2")
```

Comment:
So I had to downgrade the parquet version as there is a common version specified in settings.gradle.kts which is 1.11.2
I have updated here to make it consistent and because the newer version has some breaking changes which cause some classpath issue when we move the transformers into common.

I will see if I can remove the extra val from build.gradle.kts and have it pulled directly from the settings.gradle so all of them can be in step together and we avoid multiple version jars on the classpath.

Reply:
Moved to the settings.gralde and is now being managed from there so the common parquet dependency can be maintained.